src/App.js

```diff
@@ -16,7 +16,7 @@ function App() {
       <Routes>
         <Route path="/" element={<Doctors />} />
         <Route path="/add_doctor" element={<AddDoctor />} />
-        <Route path="/add_appointment/:id?" element={<AddReservation />} />
+        <Route path="/add_appointment/:doctorId" element={<AddReservation />} />
```

Comment:
Why did you make this change? I believe the code `:id?` was there to allow for the route to pass along information but also allow for it to be sent without a number.

Reply:
To differentiate this `:id` from:
 https://github.com/JonahKayizzi/doctors-reservations-frontend/blob/a766b7ca8eee7e8b68329569646f7a9fa251658d/src/App.js#L22
into `params` and avoid unexpected behaviors, but probably it's also unnecessary.

Reply:
Done 😄